src/app/content/components/CenteredContentRow.tsx

```diff
@@ -1,37 +1,69 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import styled, { css } from 'styled-components/macro';
 import theme from '../../theme';
+import { AppState } from '../../types';
+import * as searchSelectors from '../search/selectors';
+import { searchResultsOpen } from '../search/selectors';
+import { mobileMenuOpen, tocOpen } from '../selectors';
 import { State } from '../types';
 import { contentWrapperMaxWidth } from './constants';
+import { styleWhenSidebarClosed } from './utils/sidebar';
 
-// tslint:disable-next-line: variable-name
-const Wrapper = styled.div<{verticalNavOpen: State['tocOpen']}>`
+const collapseGridCss = css`
   grid-column: 1 / -1;
+`;
+
+const expandGridCss = css`
+  grid-column: 2;
+`;
+
+// tslint:disable-next-line: variable-name
+const Wrapper = styled.div<{isTocOpen: State['tocOpen'], isVerticalNavOpen: boolean, isDesktopSearchOpen: boolean}>`
+  ${collapseGridCss}
   grid-row: 1;
   justify-self: center;
   width: 100%;
   overflow: visible; /* so sidebar position: sticky works */
   margin: 0 auto;
   padding-left: 0;
   max-width: ${contentWrapperMaxWidth}rem;
-  ${theme.breakpoints.mobile(css`
-    grid-column-start: 2;
-  `)}
-  ${theme.breakpoints.mobileMedium(css`
-    grid-column: 1 / -1;
-  `)}
+  ${styleWhenSidebarClosed(expandGridCss)}
+  ${(props) => (props.isTocOpen || props.isTocOpen === null) &&
+    theme.breakpoints.desktop(collapseGridCss)} /* The grid column is not needed at a desktop breakpoint */
+  ${(props) => !props.isTocOpen &&
+    theme.breakpoints.mobileMedium(collapseGridCss)} /* Invert the above behavior for specific breakpoint */
 
   @media screen {
     display: flex;
     flex-direction: row;
   }
 `;
 
+type CenteredContentRowProps = React.PropsWithChildren<{
+  isDesktopSearchOpen: State['tocOpen'];
+  isTocOpen: State['tocOpen'];
+  isVerticalNavOpen: State['tocOpen'];
+}>;
+
 // tslint:disable-next-line: variable-name
-const CenteredContentRow: React.FunctionComponent = ({ children }) => {
-  return <Wrapper data-testid='centered-content-row'>
+const CenteredContentRow = ({
+  children, isTocOpen, isVerticalNavOpen, isDesktopSearchOpen,
+}: CenteredContentRowProps) => {
+  return <Wrapper
+    data-testid='centered-content-row'
+    isTocOpen={isTocOpen}
+    isVerticalNavOpen={isVerticalNavOpen}
+    isDesktopSearchOpen={isDesktopSearchOpen}
+  >
     {children}
   </Wrapper>;
 };
 
-export default CenteredContentRow;
+export default connect(
+  (state: AppState) => ({
+    isDesktopSearchOpen: !!searchSelectors.query(state),
```

Comment:
Shouldn't the right hand side use `searchSelectors.searchResultsOpen(state)` instead?